rest/config.go

```diff
@@ -165,40 +167,41 @@ func (c ClusterConfig) CBGTEnabled() bool {
+	OldRevExpirySeconds              *uint32                          `json:"old_rev_expiry_seconds,omitempty"`                  // The number of seconds before old revs are removed from CBS bucket
+	ViewQueryTimeoutSecs             *uint32                          `json:"view_query_timeout_secs,omitempty"`                 // The view query timeout in seconds
+	LocalDocExpirySecs               *uint32                          `json:"local_doc_expiry_secs,omitempty"`                   // The _local doc expiry time in seconds
+	EnableXattrs                     *bool                            `json:"enable_shared_bucket_access,omitempty"`             // Whether to use extended attributes to store _sync metadata
+	SecureCookieOverride             *bool                            `json:"session_cookie_secure,omitempty"`                   // Override cookie secure flag
+	SessionCookieName                string                           `json:"session_cookie_name"`                               // Custom per-database session cookie name
+	SessionCookieHTTPOnly            bool                             `json:"session_cookie_http_only"`                          // HTTP only cookies
+	AllowConflicts                   *bool                            `json:"allow_conflicts,omitempty"`                         // False forbids creating conflicts
+	NumIndexReplicas                 *uint                            `json:"num_index_replicas"`                                // Number of GSI index replicas used for core indexes
+	UseViews                         bool                             `json:"use_views"`                                         // Force use of views instead of GSI
+	SendWWWAuthenticateHeader        *bool                            `json:"send_www_authenticate_header,omitempty"`            // If false, disables setting of 'WWW-Authenticate' header in 401 responses
+	BucketOpTimeoutMs                *uint32                          `json:"bucket_op_timeout_ms,omitempty"`                    // How long bucket ops should block returning "operation timed out". If nil, uses GoCB default.  GoCB buckets only.
+	DeltaSync                        *DeltaSyncConfig                 `json:"delta_sync,omitempty"`                              // Config for delta sync
+	CompactIntervalDays              *float32                         `json:"compact_interval_days,omitempty"`                   // Interval between scheduled compaction runs (in days) - 0 means don't run
+	SGReplicateEnabled               *bool                            `json:"sgreplicate_enabled,omitempty"`                     // When false, node will not be assigned replications
+	SGReplicateWebsocketPingInterval *time.Duration                   `json:"sgreplicate_websocket_heartbeat_seconds,omitempty"` // If set, uses this duration as a custom heartbeat interval for websocket ping frames
```

Comment:
```suggestion
	SGReplicateWebsocketPingInterval *time.Duration                   `json:"sgreplicate_websocket_heartbeat_secs,omitempty"` // If set, uses this duration as a custom heartbeat interval for websocket ping frames
```
We use _secs elsewhere in this config, updating for consistency.